resources/content/config-content/host-iptables/apply.sh

```diff
@@ -22,4 +22,9 @@ done
 
 add_route_table 300
 
+
+if ! ip route show | grep 169.254.169.250; then
```

Comment:
This won't fly because we are hard coding docker0.  Let me look, I think there is a better place for this.